constraint/.golangci.yaml

```diff
@@ -39,3 +38,10 @@ linters:
   - typecheck
   - unused
   - whitespace
+
+issues:
+  exclude-rules:
+    - linters: [revive]
+      text: 'redefines-builtin-id: redefinition of the built-in function new'
+  exclude:
+    - 'deprecated: This package is intended for older projects transitioning from OPA v0.x and will remain for the lifetime of OPA v1.x'
```

Comment:
@charlieegan3 @anderseknert this is from https://pkg.go.dev/github.com/open-policy-agent/opa/v1 can you please confirm if switching to github.com/open-policy-agent/opa/v1/ will allow us to continue to use v0 Rego syntax without any issue?

Reply:
I think this must have been disabled since the project still imports various v0 packages in this branch.

In https://github.com/open-policy-agent/frameworks/pull/517 I have fixed these imports to the v1 versions.

Reply:
Having a closer look, it doesn't look like any of the imports have been updated to v1. As explained here int he 1.0 docs https://www.openpolicyagent.org/docs/latest/v0-upgrade/#upgrading-for-go-integrations, you can upgrade to the v1 packages and still use the v0 functionality. This is explained in more detail here:
https://www.openpolicyagent.org/docs/latest/v0-compatibility/#v0x-compatibility-mode-in-rego-package